cmd/guaccollect/cmd/deps_dev.go

```diff
@@ -114,8 +114,18 @@ you have access to read and write to the respective blob store.`,
 	},
 }
 
-func validateDepsDevFlags(pubsubAddr string, blobAddr string, csubAddr string, csubTls bool, csubTlsSkipVerify bool, useCsub bool, poll bool, retrieveDependencies bool, args []string,
-	enablePrometheus bool, prometheusPort int,
+func validateDepsDevFlags(
```

Comment:
There are some changes like this where in the full-diff view it looks like I made purely aesthetic changes but that is mostly not how it came about: originally I had added a flag to the collector commands and that required touching each one and I just tidied up a bit as I went. But when I was done I thought: maybe it make more sense to not over-complicate each of the collector commands by adding a flag to them and instead just always populate it. I ran this by @pxp928 and we discussed that this route was preferable because it left the code a bit simpler. In the in-memory backend we still populate the `documentRef` field to no real purpose, but it shouldn't hurt anything and allowed me to get rid of all of the new flags.